src/composables/multisigAccounts.ts

```diff
@@ -29,6 +29,10 @@ const POLLING_INTERVAL = 7000;
 const LOCAL_STORAGE_MULTISIG_KEY = 'multisig';
 const LOCAL_STORAGE_MULTISIG_PENDING_KEY = 'multisig-pending';
 
+export interface MultisigAccountsOptions extends IDefaultComposableOptions {
+  pollOnce?: boolean;
```

Comment:
```suggestion
  pollOnce?: boolean;
```
I'm not fully satisfied with the variable name. I'm open for suggestions

Reply:
what do you think about `fetchOnce`?

Reply:
Maybe follow the same pattern as here: `src/composables/notifications.ts`?

Reply:
It's just a little bit different. We would still load once in case it never been loaded, if it was loaded we will use state from the localstorage.